File: test_javaPrj_01/src/programmers/lv0/머쓱이보다키큰사람.java
package src.programmers.lv0;

import java.util.Arrays;

public class 머쓱이보다키큰사람 {

    private 머쓱이보다키큰사람(){}


    public void Solution() throws Exception{

        int[] array= {149, 180, 192, 170};

        int height = 167;
        
        System.out.println((int)Arrays.stream(array).filter(o -> o > height).count());
        

    }

    public static void main(String[] args) {
        
        try{
            new 머쓱이보다키큰사람().Solution();
        }catch(Exception e){
            e.printStackTrace();
        }


    }
    
}

File: test_javaPrj_01/src/programmers/lv0/머쓱이보다키큰사람.py


class 머쓱이보다키큰사람:

    def __init__(self, array:list[int], height:int) -> None:

        print( len(list(filter(lambda x: x > height, array))) )


머쓱이보다키큰사람([149, 180, 192, 170]	, 167)